waves-integration/src/main/scala/com/wavesplatform/dex/domain/transaction/ExchangeTransaction.scala

```diff
@@ -134,4 +137,45 @@ object ExchangeTransaction {
           ExchangeTransactionV3.parseBytes(bytes).map(_._1)
       }
 
+  //converts asset_decimal price to fixed_decimal
+  def convertPrice(price: Long, amountAssetDecimals: Int, priceAssetDecimals: Int): Either[GenericError, Long] =
```

Comment:
Maybe name it convertPriceToFixedDecimals?

Reply:
Comment & function arguments look quite expressive in my opinion. Anyway, it's just rollback of one of the previous versions and not worth a 45 minute flapping build.